packages/edit-site/src/components/sidebar-navigation-screen-main/index.js

```diff
@@ -31,35 +31,40 @@ export default function SidebarNavigationScreenMain() {
 		<SidebarNavigationScreen
 			isRoot
 			title={ __( 'Design' ) }
+			description={ __(
+				'Customise your Navigation menus, Templates, and more.'
```

Comment:
```suggestion
				'Customise your Navigation menus, Templates, and Styles.'
```

Reply:
I guess it's okay to begin with, but won't scale that well when we add more features. It also kind of just repeats the menu items listed below xD

We might consider something a little more abstract in the future.

> Customise the appearance of your website to match your unique vision.

Reply:
"Customize the appearance of your website using the block editor."? The "unique vision" feels too single user centered.